blocks/init/src/Blocks/custom/accordion/components/accordion-editor.js

```diff
@@ -8,7 +8,7 @@ export const AccordionEditor = ({ attributes, setAttributes }) => {
 		<AccordionEditorComponent
 			{...props('accordion', attributes, {
 				setAttributes: setAttributes,
-				accordionAccordionContent: <InnerBlocks />
+				accordionContent: <InnerBlocks />
```

Comment:
This should be ok if you are receiving 
`content` in `component/accordion` but since you are suppose to get `accordionContent` I believe this is not working as it should.

Reply:
As I explained in a comment above, becuase this is a (weird edge case and a) component, and components usually don't have `InnerBlocks`, we have to pass this directly to the component's editor, that's why it's `accordionContent` and not `accordionAccordionContent`.